types/src/v2/params.rs

```diff
@@ -335,4 +327,24 @@ mod test {
 			assert_eq!(&serialized, initial_ser);
 		}
 	}
+
+	#[test]
+	fn subscription_params_serialize_work() {
+		let ser =
+			serde_json::to_string(&JsonRpcSubscriptionParams { subscription: SubscriptionId::Num(12), result: "goal" })
+				.unwrap();
+		assert_eq!(ser, r#"{"subscription":12,"result":"goal"}"#);
```

Comment:
Super tiny nit pick, more of a consistency suggestion. In the other tests you save the expected result to `let exp` and then plug it into `assert_eq!` which I think is definitely clean. 

I just noticed here you plug the expected result directly in the test macro. Up to you but it would make it more consistent to save it to a `exp` here as well.

Reply:
fair enough, it saved me one line :)